### PR TITLE
[example] Firefox CFR PIP message

### PR DESCRIPTION
**Pull request by piatra**
image

### CODE REVIEW

**Comment by piatra** [comment]
Not sure how to correctly split targeting between experiment targeting and message targeting.

File: messaging-experiments.yaml [hunk 1]
```diff
@@ -650,3 +650,89 @@
                 - web.whatsapp.com
                 - www.messenger.com
                 - messenger.com
+- id: RECOMMEND_PICTURE_IN_PICTURE_CFR
+  enabled: true
+  arguments:
+    slug: bug-1635199-message-recommend-picture-in-picture-cfr-beta-77-77
+    branches:
+      - slug: control
```
Comment (k88hudson) [inline messaging-experiments.yaml:658]: This should be `n4s_pip_control`

File: messaging-experiments.yaml [hunk 2]
```diff
@@ -650,3 +650,89 @@
...
+        ratio: 50
+        value: {}
+        groups:
+          - cfr
+      - slug: variant_1
```
Comment (k88hudson) [inline messaging-experiments.yaml:663]: This should be `n4s_pip_cfr`

File: messaging-experiments.yaml [hunk 3]
```diff
@@ -650,3 +650,89 @@
...
+          - cfr
+        value:
+          id: N4S_PIP_CFR_MESSSAGE
+          content:
+            bucket_id: N4S_PIP_CFR_MESSSAGE
+            buttons:
+              primary:
+                action:
+                  data:
+                    args: https://support.mozilla.org/en-US/kb/about-picture-picture-firefox
```
Comment (k88hudson) [inline messaging-experiments.yaml:675]: This URL shouldn't include the locale, it will be detected

File: messaging-experiments.yaml [hunk 4]
```diff
@@ -650,3 +650,89 @@
...
+              label:
+                string_id: cfr-doorhanger-extension-sumo-link
+              sumo_path: extensionrecommendations
+            layout: icon_and_message
+            notification_text:
+              string_id: cfr-doorhanger-extension-notification
+            text: Pop video into a floating window to keep watching while you work in other tabs. Hover over any video and select the small blue rectangle that appears.
+          frequency:
+            lifetime: 3
+          targeting: 'true'
```
Comment (k88hudson) [inline messaging-experiments.yaml:708]: I think omitting this is fine, messages without targeting automatically match

File: messaging-experiments.yaml [hunk 5]
```diff
@@ -650,3 +650,89 @@
...
+              - www.ustream.com
+              - www.metacafe.com
+              - www.espn.com
+              - www.disneyplus.com
+    isHighVolume: false
+    userFacingName: Message Router Content Experiment Recommend Picture-in-Picture CFR
+    isEnrollmentPaused: false
+    experimentDocumentUrl: https://experimenter.services.mozilla.com/experiments/recommend-picture-in-picture-cfr/
+    userFacingDescription: If we introduce PiP via CFR for users visiting video sites frequently, then we will see more adoption of the PiP feature because we have observed the feature is not easily discovered via user research observations.
+  filter_expression: localeLanguageCode == 'en' && userPrefs.cfrFeatures && firefoxVersion == 77 && [userId, 'RECOMMEND_PICTURE_IN_PICTURE_CFR']|stableSample(0.05)
```
Comment (k88hudson) [inline messaging-experiments.yaml:738]: This should be `targeting` since this is messaging system targeting

File: messaging-experiments.yaml [hunk 6]
```diff
@@ -650,3 +650,89 @@
...
+            params:
+              - youtube.com
+              - netflix.com
+              - vimeo.com
+              - twitch.com
+              - dailymotion.com
+              - hulu.com
+              - ustream.com
+              - metacafe.com
+              - espn.com
```
Comment (k88hudson) [inline messaging-experiments.yaml:721]: Does this need to be `/watch`?

File: messaging-experiments.yaml (hunk 5 above)
Comment (k88hudson) [reply messaging-experiments.yaml:738]: This should be `en-US`

Comment (k88hudson) [reply messaging-experiments.yaml:738]: This is missing `beta` channel

File: messaging-experiments.yaml [hunk 7]
```diff
@@ -650,3 +650,89 @@
...
+              - www.vimeo.com
+              - www.twitch.com
+              - www.dailymotion.com
+              - www.hulu.com
+              - www.ustream.com
+              - www.metacafe.com
+              - www.espn.com
+              - www.disneyplus.com
+    isHighVolume: false
+    userFacingName: Message Router Content Experiment Recommend Picture-in-Picture CFR
```
Comment (k88hudson) [inline messaging-experiments.yaml:734]: Should be `Recommend Picture-in-Picture CFR`

File: messaging-experiments.yaml (hunk 5 above)
Comment (k88hudson) [reply messaging-experiments.yaml:738]: Should use `bucketSample` with `n4s-workweek` as the namespace, starting at `0`

Comment (tdsmith) [reply messaging-experiments.yaml:738]: not critical for English, but `localeLanguageCode == 'en'` doesn't look like the same thing as the `en-US` spec in Experimenter

**Pull request merged**